andela/vale-ah-backend
# #164139779 following users

#### What does this PR do?
-Follow users
-unfollow users
#### Description of Task to be completed?
- write unit test
- Create a route for following a user
- Create a route for unfollowing a user
-Create a route for viewing a user's followers
-Create a route for viewing users' followee
- validate inputs with `joi`
#### How should this be manually tested?
- ensure that the build is passing on travis 
- Test endpoint api/user/follower
- Test endpoint api/user/following
- Test endpoint api/profiles/:username/follow
- Test endpoint api/profiles/:username/unfollow

#### What are the relevant pivotal tracker stories?
[#164139779](https://www.pivotaltracker.com/story/show/164139779)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
